astropy/timeseries/bls/core.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 
 from astropy import units
-from astropy.stats.lombscargle.core import has_units, strip_units
+from astropy.timeseries.lombscargle.core import has_units, strip_units
```

Comment:
I think it would make sense if these were refactored out into a `timeseries.periodogram.utils` (but not in this PR)

Reply:
Ok sounds good